tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -611,12 +617,12 @@ cc_library(
         "//tensorflow/core:stream_executor_no_cuda",
         "//tensorflow/core/util/proto:proto_utils",
         "//tensorflow/stream_executor:device_memory_allocator",
-        "//tensorflow/stream_executor/cuda:redzone_allocator",
         "@com_google_absl//absl/algorithm:container",
         "@com_google_absl//absl/strings",
         "@com_google_absl//absl/strings:str_format",
         "@com_google_absl//absl/time",
         "@com_google_absl//absl/types:optional",
+        "//tensorflow/stream_executor/cuda:redzone_allocator",
```

Comment:
FYI updated dependency location because `redzone_allocator` will become platform dependent and guarded by `if_cuda_is_configured` in the next PR.